tests/mock.js

```diff
@@ -23,25 +22,34 @@ class TestMock {
     if(!this.keys) {
       // create mock keys
       this.keys = {};
-
-      // create KEK and HMAC keys for creating data hubs
-      this.keys.kek = await MockKek.create();
+      // this creates the same invocationSigner for each test.
+      this.invocationSigner = new MockInvoker();
+      // create KAK and HMAC keys for creating data hubs
+      // this creates the same keyAgreementKey for each test.
+      this.keys.keyAgreementKey = new MockKaK();
+      // the creates the same hmac for each test.
       this.keys.hmac = await MockHmac.create();
+      this.keyResolver = ({id}) => {
+        if(this.keys.keyAgreementKey.id === id) {
+          return this.keys.keyAgreementKey;
+        }
+        return this.keys.hmac;
```

Comment:
Why is this here? It seems like `throw new Error(`Key "${id}" not found.`);` would be better. My memory is that `keyResolver` functions are only for kaks, but I could be mistaken.

Reply:
I actually just left that as is, but yeah as far as I can tell it's only for Kaks.

Reply:
Addressed in https://github.com/digitalbazaar/secure-data-hub-client/pull/10/commits/be488030528e73efdf8f319b8d61e31036c59a7d

and I believe that is it.